sky/provision/gcp/instance_utils.py

```diff
@@ -124,6 +124,7 @@ def stop(
             project=project_id,
             zone=zone,
             instance=instance,
+            discardLocalSsd=False,
```

Comment:
nit: Shall we add a comment for why we need this argument? Maybe add a reference to doc of this method too: https://cloud.google.com/compute/docs/reference/rest/v1/instances/stop